File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/SchoolingInfoDto.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

import org.taktik.icure.entities.base.CodeStub;

import java.io.Serializable;

public class SchoolingInfoDto implements Serializable {
    private Long startDate;
    private Long endDate;
    private String school;
    private CodeStub typeOfEducation;

    public Long getStartDate() {
        return startDate;
    }

    public void setStartDate(Long startDate) {
        this.startDate = startDate;
    }

    public Long getEndDate() {
        return endDate;
    }

    public void setEndDate(Long endDate) {
        this.endDate = endDate;
    }

    public String getSchool() {
        return school;
    }

    public void setSchool(String school) {
        this.school = school;
    }

    public CodeStub getTypeOfEducation() {
        return typeOfEducation;
    }

    public void setTypeOfEducation(CodeStub typeOfEducation) {
        this.typeOfEducation = typeOfEducation;
    }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/SchoolingInfoDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import org.taktik.icure.services.external.rest.v1.dto.base.CodeStubDto
import java.io.Serializable
import com.github.pozo.KotlinBuilder
import com.github.pozo.KotlinBuilder
@KotlinBuilder
data class SchoolingInfoDto(
        val startDate: Long? = null,
        val endDate: Long? = null,
        val school: String? = null,
        val typeOfEducation: CodeStubDto? = null
) : Serializable
